packages/forklift-console-plugin/src/components/mappings/MappingRow.tsx

```diff
@@ -67,6 +73,18 @@ const NameCell: React.FC<CellProps<CommonMapping>> = ({ resourceData }) => {
   );
 };
 
+const ReadyCell: React.FC<CellProps<CommonMapping>> = ({ resourceData }) => {
+  const { t } = useTranslation();
+
+  return (
+    <StatusCell
+      conditions={resourceData.conditions}
+      icon={resourceData.ready ? <GreenCheckCircleIcon /> : <RedExclamationCircleIcon />}
+      label={resourceData.ready ? t('Yes') : t('No')}
```

Comment:
```suggestion
      label={resourceData.ready ? t('Ready') : t('Not ready')}
```
I actually want a status column, but we don't have phase for this CRD, best next thing is to show the ready condition as status until we add a phase in the backend